macros/SVGAttr.ejs

```diff
@@ -3,7 +3,6 @@
 var slug = mdn.localString({
     "en-US": "Attribute",
     "de"   : "Attribut",
```

Comment:
The same applies to `de`: [`/de/docs/Web/SVG/Attribute`](https://developer.mozilla.org/de/docs/Web/SVG/Attribute)